lib/katello/permission_creator.rb

```diff
@@ -449,5 +451,34 @@ def organization_permissions
                          },
                          :resource_type => 'Organization'
     end
+
+    def flatpak_remote_permissions
+      @plugin.permission :view_flatpak_remotes,
+                         {
+                           'katello/api/v2/flatpak_remotes' => [:index, :show, :auto_complete_search],
+                           'katello/api/v2/flatpak_remote_repositories' => [:index, :show, :auto_complete_search],
```

Comment:
For some reason, my user with view_flatpak_remotes can't view an individual remote repository. They can however view the listing page for remotes and remote repositories. They can also view individual remotes.

Reply:
Fixed!